index.browser.js

```diff
@@ -47,25 +47,15 @@ export let customRandom = (alphabet, defaultSize, getRandom) => {
 export let customAlphabet = (alphabet, size = 21) =>
   customRandom(alphabet, size, random)
 
-export let nanoid = (size = 21) =>
-  crypto.getRandomValues(new Uint8Array(size)).reduce((id, byte) => {
-    // It is incorrect to use bytes exceeding the alphabet size.
-    // The following mask reduces the random byte in the 0-255 value
-    // range to the 0-63 value range. Therefore, adding hacks, such
-    // as empty string fallback or magic numbers, is unneccessary because
-    // the bitmask trims bytes down to the alphabet size.
-    byte &= 63
-    if (byte < 36) {
-      // `0-9a-z`
-      id += byte.toString(36)
-    } else if (byte < 62) {
-      // `A-Z`
-      id += (byte - 26).toString(36).toUpperCase()
-    } else if (byte > 62) {
-      id += '-'
-    } else {
-      id += '_'
-    }
-    return id
-  }, '')
-
+export let nanoid = (len = 21) => {
+	let id = "";
```

Comment:
1. Let’s fix code style issues from ESLint.
2. And we will need to update `package→size-limit` to see how much we increase the JS bundle size (for browser’s version we are optimizing both size and performance)